src/templates/healthcert/parsers/vacHealthCertV1/generateMemoSections.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { pdtHealthCertV1 as pdtHealthcert } from "@govtechsg/oa-schemata";
+import { NotarisedHealthCert, Patient, Immunization, ImmunizationRecommendation, Location } from "../../types";
+import { MemoSection } from "../../memo/vacV1Memo";
+import { formatDate, simplifyImmunizationObjectWithLocation } from "./parseInfo";
+
+const isNric = (value: pdtHealthcert.Identifier): boolean =>
+  typeof value.type !== "string" && value.type.text === "NRIC";
+
+export const generateMemoSections = (document: NotarisedHealthCert, multiQr = false): JSX.Element[] => {
+  const patient = document.fhirBundle.entry.find(entry => entry.resourceType === "Patient") as Patient;
+  const locations = document.fhirBundle.entry.filter(entry => entry.resourceType === "Location") as Location[];
+  const immunizations = document.fhirBundle.entry.filter(
+    entry => entry.resourceType === "Immunization"
+  ) as Immunization[];
+  const recommendation = document.fhirBundle.entry.find(
+    entry => entry.resourceType === "ImmunizationRecommendation"
+  ) as ImmunizationRecommendation;
+
+  const passportNumber = document.notarisationMetadata?.passportNumber;
+  const patientName = typeof patient?.name?.[0] === "object" ? patient?.name[0]?.text : "";
+  const patientNric = patient?.identifier?.find(isNric)?.value || "";
+  const patientNationalityCode =
+    patient?.extension?.find(
+      extension => extension.url === "http://hl7.org/fhir/StructureDefinition/patient-nationality"
+    )?.code.text || "";
+  const patientBirthDate = formatDate(patient?.birthDate || "");
```

Comment:
Let's import the `isoToDateOnlyString()` function from here: https://github.com/Notarise-gov-sg/healthcert-renderer/blob/master/src/util/datetime.ts#L26-L38